apps/designer/app/designer/features/sidebar-left/panels/components/uploading-animation.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import { useInterval } from "react-use";
+
+const useFakeProgress = ({
+  isUploading,
+  isDeleting,
+}: {
+  isUploading: boolean;
+  isDeleting: boolean;
+}) => {
+  const [progressBarPercentage, setProgressBarPercentage] = useState(0);
+
+  // @todo rewrite this fake indication to show real progress
+  useInterval(
+    () => {
+      setProgressBarPercentage((percentage) =>
+        percentage < 100 ? percentage + 1 : percentage
+      );
+    },
+    isUploading || isDeleting ? 100 : null
+  );
+
+  return progressBarPercentage;
+};
+
+export const UploadingAnimation = ({
```

Comment:
Generally there is quite a bit of accessibility stuff baked in in radix's progress bar

https://github.com/radix-ui/primitives/blob/main/packages/react/progress/src/Progress.tsx